### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by mircohaug**
I translated a lot of the old dokuwiki entries via pandoc to github flavoured markdown and fixed all the broken internal and external links

### CODE REVIEW

**Comment by luoe** [comment]
I don't know why but 'ue' became 'ü' (e.g. nachzuschaün) and other things also became Umlaute ('ae' etc.). But it looks great so far!

File: infos/anfi-faq.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ genau solche Fragen Ansprechpartner ist. Was aber nicht heißen muss,
 dass wir alles wissen bzw. dass wir auch immer alles korrekt wissen.
 Generell gilt daher immer, sich von offizieller Seite absichern, also in
 Vorlesungsverzeichnissen, Studienordnungen, auf Schwarzen Brettern, ...
-nachzuschauen und bei den
+nachzuschaün und bei den
```
Comment (luoe) [inline infos/anfi-faq.md]: For example this

**Comment by mircohaug** [comment]
Contains links which are only valid after https://github.com/fsi-tue/anfiheft/pull/1 and https://github.com/fsi-tue/anfibrief/pull/26 are merged

File: infos/anfi-faq.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Er ist nicht zwingend erforderlich, aber sehr nützlich, wenn man sich
 mit Mathe sowieso etwas schwer tut. Er fängt 2,5 Wochen vor dem Semester
 an und dauert zwei Wochen. Eine Anmeldung ist hierfür nicht
 erforderlich, nähere Informationen entnehmt ihr am besten dem
-[Anfängerbrief]().  
+[Anfängerbrief](https://github.com/fsi-tue/anfibrief/tree/master/pdf) oder dem Anfängerheft für [*Informatiker](https://github.com/fsi-tue/anfiheft/blob/master/anfiheft-info.pdf) oder [Kognis](https://github.com/fsi-tue/anfiheft/blob/master/anfiheft-kogni.pdf).
```
Comment (luoe) [inline infos/anfi-faq.md]: You should replace the links for Anfibrief pointing to the PDFs with the ones hosted on our infrastructure. See https://teri.fsi.uni-tuebingen.de/anfibrief/ . 
For now we can use the link https://www.fsi.uni-tuebingen.de/erstsemester/material/start for Anfiheft, we change the link in the future when we have CI/CD for Anfiheft so it is also hosted on https://teri.fsi.uni-tuebingen.de

**Pull request merged**